src/main/java/gov/nih/nci/bento/error/BentoGraphqlError.java

```diff
@@ -0,0 +1,43 @@
+package gov.nih.nci.bento.error;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BentoGraphqlError {
```

Comment:
Other project will use this class too. I think we should better rename it without Bento